worker-message-prioritization-target-refill/src/main/java/com/github/workerframework/workermessageprioritization/targetrefill/K8sTargetQueueRefillProvider.java

```diff
@@ -0,0 +1,155 @@
+/*
```

Comment:
I'm guessing this is very similar to other code that's retrieving labels from k8s.
If labels will continue to be part of the solution to US692066, we want to avoid this code duplication and also the additional load on K8S caused by 2 different processes retrieving labels.

Reply:
@andyreidz , yes have left comment above that this needs to be changed.